### ISSUE TITLE
http://remote-tilt.com/tilt.html issue

### ISSUE THREAD

**Issue by andreasbovens** [remy/remote-tilt#4]
There seems to be something wrong with the http://remote-tilt.com/tilt.html demo.
1. Go to http://remote-tilt.com/tilt.html
2. For "To use a real mobile device for motion events or you can't enable popups, select 'OK' to continue. Or, select 'cancel' and enable popups to use the mini remote." choose OK. 
3. Hit OK for "Visiting http://remote-tilt.com/MY_REMOTE_KEY will remotely send motion events to this page"

However, visiting http://remote-tilt.com/MY_REMOTE_KEY with any browser with orientation event support (incl. Safari on iPhone) does not seem to result in the expected effect of the logo being controlled by the orientation of the device. Strangely enough, what did happen was that the orientation events of my MacBook Pro were captured and they were steering the orientation of the logo...

So, in short, pressing "Cancel" or "OK" in step 2 gives the same results.

**Comment by remy** [comment]
Hmm - just re-tested this and it's working fine for me. Trying to think what might be causing the issue...

The events on your laptop should still be able to control the events, but the popup should _also_ be able to control the events.  

Does the popup send any of the events to the test page (tilt.html)?

**Comment by remy** [comment]
Actually - I just found that my server was playing up.  I'm doubtful, but it's possibly that was causing the issue.  Want to try again?
